PCSX2_suite.sln

```diff
@@ -70,352 +70,720 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "d3d12memalloc", "3rdparty\d
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "lzma", "3rdparty\lzma\lzma.vcxproj", "{A4323327-3F2B-4271-83D9-7F9A3C66B6B2}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "liblz4", "3rdparty\lz4\build\VS2017\liblz4\liblz4.vcxproj", "{9092C5CC-3E71-41B3-BF68-4A7BDD8A5476}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug AVX2|x64 = Debug AVX2|x64
+		Debug AVX2|x86 = Debug AVX2|x86
 		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
 		Devel AVX2|x64 = Devel AVX2|x64
+		Devel AVX2|x86 = Devel AVX2|x86
 		Devel|x64 = Devel|x64
+		Devel|x86 = Devel|x86
 		Release AVX2|x64 = Release AVX2|x64
+		Release AVX2|x86 = Release AVX2|x86
 		Release|x64 = Release|x64
+		Release|x86 = Release|x86
```

Comment:
This whole file looks like a messed up merge
These look like they're from when we had an x86 build
And below looks like you duplicated everything

Reply:
Thanks for the feedback! It wasn't a merge. I followed the instructions here to build  https://wiki.pcsx2.net/PCSX2_Documentation/Compiling_on_Windows only to find out all the NIS files were removed and just use cmake. Do you have a suggestion?

Reply:
You don't _need_ cmake on windows. Just double click the PCSX2_suite.sln or PCSX2_qt.sln

Reply:
Right then what? Are there new build instructions?

Reply:
![image](https://user-images.githubusercontent.com/29295048/178116599-e7015515-ba39-44ed-8ea0-49e763442651.png)

Reply:
Regardless of how it got here, you do need to fix it

Reply:
Yep I'm on it. Thanks.